registry_inspector/tests/test_registry_client.py

```diff
@@ -94,3 +94,11 @@ def headers_callback(request):
 
     resp = client.get_digest_length(name, digest)
     assert resp == '32'
+
+
+def test_log(capsys):
+    registry = 'http://localhost:5000'
+    client = RegistryClient(registry)
+    client.log("log testing")
+    out, err = capsys.readouterr()
+    assert err == "log testing\n"
```

Comment:
You probably want to assert on the value of `out` as well. Maybe `assert (out, err) == ("", "...")` or something?